lib/westfield_datetime_helper/validators/zulu_time.rb

```diff
@@ -0,0 +1,35 @@
+# Validates that a supplied value is in fact
+# a DateTime, and is in timezone UTC+0
+require 'time'
+require 'active_model'
+
+module WestfieldLabs
+  module DatetimeHelper
+    module Validators
+      class ZuluTimeValidator < ActiveModel::EachValidator
+
+        def validate_each(record, attribute, value)
+          return if value.is_a? DateTime and utc_datetime? value
+          return if value.is_a? Time and utc_time? value
+          return if value.is_a? String and zulu_time? value
```

Comment:
Alternatively, should this accept strings at all? Is there a use-case for saving dates in a string field, rather than a DateTime or Time field?